src/components/SidebarMobile/SidebarMobile.jsx

```diff
@@ -37,18 +39,9 @@ export default class SidebarMobile extends React.Component {
     );
   }
 
-  componentDidMount() {
-    if (typeof window !== 'undefined') {
-      window.addEventListener('mousedown', this._handleBodyClick);
-      // window.addEventListener('touchstart', this._handleBodyClick);
-    }
-  }
-
-  componentWillUnmount() {
-    if (typeof window !== 'undefined') {
-      window.removeEventListener('mousedown', this._handleBodyClick);
-      // window.removeEventListener('touchstart', this._handleBodyClick);
-    }
+  _toggleBodyListener(add) {
+    let actionName = add ? 'addEventListener' : 'removeEventListener';
+    window[actionName]('mousedown', this._handleBodyClick);
```

Comment:
Did it build?

Reply:
yes

Reply:
if you mean `npm run build`

Reply:
oh yeah, i was confused with something, nvm